tests/test_krylov.py

```diff
@@ -0,0 +1,99 @@
+import scipy
+import tequila as tq
+from tequila import braket
+from tequila.hamiltonian.qubit_hamiltonian import QubitHamiltonian
+from tequila.tools.random_generators import make_random_circuit
+import itertools as it
+import numpy as np
+
+def simple_krylov(n_krylov_states: int=2):
```

Comment:
needs to be renamed to test_simple_krylov otherwise the test-runner will not pick it up